portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+public final class Comment {
+
+    private final String name;
```

Comment:
Nice to make these immutable once created.

Reply:
:)